src/entities/Schedule/model.ts

```diff
@@ -1,43 +1,28 @@
 import Time from "decentraland-gatsby/dist/utils/date/Time"
+import { Model } from "decentraland-gatsby/dist/entities/Database/model"
 import { ScheduleAttributes } from "./types"
+import { SQL, table } from "decentraland-gatsby/dist/entities/Database/utils"
 
-const mock: ScheduleAttributes[] = [
-  {
-    id: "ea869a5a-4201-45d9-beda-7ffb07f8a88a",
-    name: "Pride Month",
-    description: "Join us for pride month on Decentraland.",
-    background: [
-      "rgba(255, 188, 91, .1)",
-      "rgba(252, 153, 101, .1)",
-      "rgba(255, 116, 57, .1)",
-      "rgba(255, 46, 84, .1)",
-      "rgba(198, 64, 205, .1)",
-      "rgba(165, 36, 179, .1)",
-      "rgba(105, 31, 169, .1)",
-    ],
-    image:
-      "https://decentraland.org/blog/images/static/images/pride-4be984a59111e68fb428a3ec060c2f07.jpg",
-    active_since: Time.from("2022-06-01 00:00:00").toDate(),
-    active_until: Time.from("2022-07-01 00:00:00").toDate(),
-  },
-]
+export default class ScheduleModel extends Model<ScheduleAttributes> {
+  static tableName = "schedule"
+  static primaryKey = "id"
 
-export default class ScheduleModel {
   static async getSchedules() {
     const now = Time.from()
-    return mock.filter((schedule) => now.isBefore(schedule.active_until))
-  }
-
-  static async findOne(filters: { id: string }) {
-    return mock.find((schedule) => schedule.id === filters.id) || null
-  }
+    const query = SQL`SELECT * FROM ${table(
+      ScheduleModel
```

Comment:
```suggestion
      this
```

use `this` when ever you are making reference to the current class